stingray/modeling/tests/test_parameterestimation.py

```diff
@@ -53,7 +53,8 @@ def setup_class(cls):
             scipy.stats.norm(loc=cls.a_mean, scale=cls.a_var).pdf(amplitude)
 
         cls.priors = {"amplitude": p_amplitude}
-        cls.lpost = PSDPosterior(cls.ps, cls.model)
+        cls.lpost = PSDPosterior(cls.ps.freq, cls.ps.power, cls.model,
```

Comment:
I passed them explicitly to be consistent with the behaviour of the superclass and all other implementations of subclasses of `LogLikelihood` and `Posterior`. 

I always get annoyed by packages like matplotlib that are really inconsistent in how they pass certain parameters, so I figured I wanted to avoid that.